nupic/algorithms/anomaly.py

```diff
@@ -123,25 +126,9 @@ def compute(self, activeColumns, predictedColumns,
                               	(used in anomaly-likelihood)
     @return the computed anomaly score; float 0..1
```

Comment:
Yes, revert. That is a pretty invasive change and we would want to coordinate the change. I personally don't see a benefit from it. Plus it should be in a separate PR regardless.